pkg/controller/utils/errorscrub.go

```diff
@@ -5,7 +5,8 @@ import (
 )
 
 var (
-	newlineTabRE = regexp.MustCompile(`\n\t`)
+	newlineTabRE           = regexp.MustCompile(`\n\t`)
+	certificateTimeErrorRE = regexp.MustCompile(`(: current time .* is after .*)`)
```

Comment:
nit: The parens should be unnecessary (you're not using the capture group).

I'm also a little concerned about the broadness of the `.*`s -- I've seen regexes like this unintentionally match huge chunks of a string because something _way later_ happens to also say "is after". Perhaps at least match non-whitespace for the timestamps?
```suggestion
	certificateTimeErrorRE = regexp.MustCompile(`: current time \S+ is after \S+`)
```